gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/EhrExtractHandlingIT.java

```diff
@@ -43,8 +43,11 @@
     private static final String NHS_NUMBER_PLACEHOLDER = "{{nhsNumber}}";
     private static final List<String> IGNORED_JSON_PATHS = List.of(
         "id",
-        "entry[0].resource.id",
```

Comment:
Do you think we could list all the specific entries to ignore rather than ignore [*]? Most of IDs use the value from HL7 XML.